website/content/docs/secrets/rabbitmq.mdx

```diff
@@ -91,6 +108,21 @@ the proper permission, it can generate credentials.
     such that trusted operators can manage the role definitions, and both users
     and applications are restricted in the credentials they are allowed to read.
 
+1.  Alternatively, request credentials for a static role by reading from the
```

Comment:
```suggestion
Alternatively, request credentials for a static role by reading from the
```

Missed this one in the first review, apologies. No need to number steps if there's only one step :)